Sources/Fluent/Schema/Schema+Creator.swift

```diff
@@ -128,12 +128,13 @@ extension Schema {
 
         public func parent<E: Entity>(
             _ entity: E.Type = E.self,
+            name: String? = nil,
             optional: Bool = false,
             unique: Bool = false,
             default: NodeRepresentable? = nil
         ) {
             fields += Field(
-                name: "\(entity.name)_id",
+                name: name ?? "\(entity.name)_id",
```

Comment:
should this also be `\(name)_id` or should the external label perhaps be `customId: String?` or perhaps a secondary function of some sort. Just thinking out loud.